scripts/diff-features.js

```diff
@@ -127,36 +133,38 @@ function printMarkdown({ added, removed }) {
   }
 }
 
-const { argv } = yargs.command(
-  '$0 [ref1] [ref2]',
-  'Compare the set of features at refA and refB',
-  yargs => {
-    yargs
-      .positional('ref1', {
-        description: 'A Git ref (branch, tag, or commit)',
-        defaultDescription: 'ref1^',
-      })
-      .positional('ref2', {
-        description: 'A Git ref (branch, tag, or commit)',
-        defaultDescription: 'HEAD',
-      })
-      .option('format', {
-        type: 'string',
-        nargs: 1,
-        choices: ['json', 'markdown'],
-        demand: 'a named format is required',
-        default: 'markdown',
-      })
-      .option('no-github', {
-        type: 'boolean',
-        description: "Don't fetch artifacts from GitHub.",
-      })
-      .example('$0', 'compare HEAD to parent commmit')
-      .example('$0 176d4ed', 'compare 176d4ed to its parent commmit')
-      .example('$0 topic-branch main', 'compare a branch to main');
-  },
-);
-
 if (require.main === module) {
+  const { argv } = yargs.command(
```

Comment:
I've moved the yargs stuff into the `require.main == module` gating so that this module could be `required` without clobbering yargs in the `require`-ing module.